pkg/storage/storage.go

```diff
@@ -222,10 +212,14 @@ func ListUnmounted(client *occlient.Client, applicationName string) ([]StorageIn
 			}
 			storageName := getStorageFromPVC(&pvc)
 			storageSize := pvc.Spec.Resources.Requests[corev1.ResourceStorage]
-			storageList = append(storageList, StorageInfo{
-				Name: storageName,
-				Size: storageSize.String(),
-				Path: "",
+			storageList = append(storageList, Storage{
```

Comment:
can we have a `NewStorage` or `NewMachineReadableStructForStorage` to hide the meta and spec here?